### DISCUSSION TITLE
Adoptium JDK July 2021 x86

### DISCUSSION THREAD

**Discussion by Shoggi1105** [adoptium/adoptium-support#342] [Q&A] [answered]
Dear Team,

we are after Cyber Essentials and have to patch our servers inside of 14 day to get security complaint.

The security advisory was raised and we already reached the 14 days and only x64 for Windows was released.

adoptium.net

The banner give us hope that it happens:
3rd August 2021 We are creating the July 2021 PSU binaries for Eclipse Temurin 8, 11 and 16.
Primary platform binaries (Windows, Linux and MacOS on x64), with Docker images and Linux packages still to go.
Other platforms will appear as they are ready over the coming days!

But we need to plan stuff and third party also have to test their backend systems with the updatd version before we can patch our servers.

Do we have any estimates for the x86 please?

Lukas

**Comment by karianna** [comment, marked as answer]
Possibly by the end of next week, the week after that should be the worst-case scenario.  Windows x86 is a challenging platform to continue to maintain, but it is being worked on.

**Reply by Shoggi1105** [reply]
Thank you for quick update

**Reply by oooforum** [reply]
Thanks for that because a lot of software needs this build.

**Reply by Shoggi1105** [reply]
@karianna , Can we have an update if any hope is for this week to get the 16.x x86?

Regards
Lukas

**Reply by karianna** [reply]
https://github.com/adoptium/adoptium/issues/68 is the one you want to track

**Reply by karianna** [reply]
This was resolved
